js/show_contributors.js

```diff
@@ -0,0 +1,50 @@
+/**
+Shows a list of the contributors to a Phobos module
+It queries the backend contribs.dlang.io
+Backend source: https://github.com/wilzbach/phobos-contribs
+
+License:   http://boost.org/LICENSE_1_0.txt, Boost License 1.0
+
+Authors:   Sebastian Wilzbach
+*/
+
+// find out whether the current page is a module or package
+function isPackage()
+{
+    var ret = false;
+    $('.tip.smallprint').find('a').each(function(i, el) {
+        if (el.href.indexOf("package.d") >= 0)
+            ret = true;
+    });
+    return ret;
+}
+
+$(document).ready(function()
```

Comment:
I think it's good to be explicit here as people not so familiar with jQuery can immediately grasp that this callback waits for the page to be ready - other dlang javascripts also use this style (e.g. run.js)